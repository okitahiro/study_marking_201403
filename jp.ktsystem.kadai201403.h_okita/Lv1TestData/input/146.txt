[{"date":"aaaaaaaa","start":"0900","end":"1800"}]