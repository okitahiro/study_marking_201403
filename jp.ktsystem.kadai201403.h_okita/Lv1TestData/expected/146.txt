[
INVALID_DAY_STRING